<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>start</string>
    </property>
    <addaction name="start_open"/>
    <addaction name="start_Save"/>
    <addaction name="start_Save_As"/>
    <addaction name="separator"/>
    <addaction name="actionprint"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menuhelp_H">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuedit">
    <property name="title">
     <string>edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actioncut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionGet_time"/>
   </widget>
   <widget class="QMenu" name="menuformat">
    <property name="title">
     <string>format</string>
    </property>
    <addaction name="actiontypeface"/>
    <addaction name="actioncolor"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuedit"/>
   <addaction name="menuformat"/>
   <addaction name="menuhelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="start_new">
   <property name="text">
    <string>New(&amp;N)</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="start_Save_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="start_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="start_Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actiontypeface">
   <property name="text">
    <string>Typeface</string>
   </property>
  </action>
  <action name="actioncolor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionprint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionGet_time">
   <property name="text">
    <string>Get time</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
